utils/queries.tsx

```diff
@@ -29,7 +29,7 @@ export interface FetchedUser {
 }
 export const buildUserFetcher = (user: User) => {
   return async () => {
-    await user.getData();
+    await user.getData(true);
```

Comment:
Oh true, should we do this for all fetchers?

Reply:
Probably yes, since now it's pretty freaking hard to abuse this and burn the database to the ground :)